beartype/_util/text/utiltextmunge.py

```diff
@@ -3,19 +3,20 @@
 # Copyright (c) 2014-2021 Cecil Curry.
 # See "LICENSE" for further details.
 
-'''
+"""
```

Comment:
> Was that an IDE thing? I recall PyCharm being fairly opinionated about that sort of syntactic style

Hahaha, It was a PyCharm DE thing and it is very persistent about the "Triple-quote-this-or-it-shall-be-forever-underlined". There's probably a setting somewhere nested within the settings jungle gym. I can dig around and see what I can't tune, twist, poke, and otherwise perform the classic [percussive maintenice](https://tvtropes.org/pmwiki/pmwiki.php/Main/PercussiveMaintenance)

> but I'm a [fanatical Vim lackey myself](https://github.com/leycec/vimrc). Eternal [RSI](https://en.wikipedia.org/wiki/Repetitive_strain_injury) to the idolatrous non-believers!

I use Vim at work, I like the mobility, and seasoned Vim users are not to be scoffed at. Alas, I am not currently a seasoned Vim user but [I'm getting better](https://youtu.be/uBxMPqxJGqI?t=15). Yes, yes, practice makes perfect which is why I'm using the 99% chocolate of IDEs - the informal education I'm pursuing is based in PyCharm. Once completed, I shall return to the pleasant minimalist yet customizable ways of Vim